src/sbd.sysconfig

```diff
@@ -23,14 +23,22 @@ SBD_PACEMAKER=yes
 #
 SBD_STARTMODE=always
 
-## Type: yesno
+## Type: yesno / integer
 ## Default: no
 #
 # Whether to delay after starting sbd on boot for "msgwait" seconds.
 # This may be necessary if your cluster nodes reboot so fast that the
 # other nodes are still waiting in the fence acknowledgement phase.
 # This is an occasional issue with virtual machines.
 #
+# This can be either enabled by configuring a specific delay value, in
+# seconds, if a different delay than the default, "msgwait", is needed.
```

Comment:
English is not my native language but this sounds strange to me.
Would rather expect something like "This can as well be enabled by ..."

Reply:
I guess it should be probably alright although I'm not a native speaker either ;-) I wouldn't like it to sound strange to any audiences though. "The can also be enabled" should probably be more natural.